.npmignore

```diff
@@ -15,8 +15,19 @@ node_modules/npm-registry-mock
 /coverage/
 
 # don't need these in the npm package.
+Makefile
+.eslintrc.json
+.licensee.json
+.mailmap
+changelogs/*
+configure
+make.bat
```

Comment:
Had to check to make sure that the `install.sh` isn't calling `./configure` or `make`, but these are fine to leave out.